components/AnimationTiming/examples/supplemental/AnimationTimingExampleSupplemental.m

```diff
@@ -14,9 +14,8 @@
  limitations under the License.
  */
 
-#import <Foundation/Foundation.h>
```

Comment:
still need this.
in fact we need CoreGraphics too since we are using CGSize CGFloat et al